src/Ingredient.tsx

```diff
@@ -0,0 +1,11 @@
+import { TIngredient } from "./App";
+
+type IngredientProp = { ingredient: TIngredient }
```

Comment:
This name isn't incorrect, but, for consistency, it would probably make more sense to name it `TIngredientProps`